src/compas/_os.py

```diff
@@ -221,13 +221,15 @@ def prepare_environment(env=None):
         env = os.environ.copy()
 
     if PYTHON_DIRECTORY:
-        lib_bin = os.path.join(PYTHON_DIRECTORY, 'Library', 'bin')
-        if os.path.exists(lib_bin) and lib_bin not in env['PATH']:
-            env['PATH'] += os.pathsep + lib_bin
-
-        lib_bin = os.path.join(PYTHON_DIRECTORY, 'lib')
-        if os.path.exists(lib_bin) and lib_bin not in env['PATH']:
-            env['PATH'] += os.pathsep + lib_bin
+        if is_windows():
+            lib_bin = os.path.join(PYTHON_DIRECTORY, 'Library', 'bin')
+            if os.path.exists(lib_bin) and lib_bin not in env['PATH']:
+                env['PATH'] = lib_bin + os.pathsep + env['PATH']
+
+        else:
+            lib_bin = os.path.join(PYTHON_DIRECTORY, 'bin')
```

Comment:
So, `os.path.join(PYTHON_DIRECTORY, 'lib')` is not needed?

Reply:
on windows it doesn't exist and the equivalent of the `Library/bin` folder on mac is `bin`. on the command line none of the lib folders are added...